spk/tt-rss/Makefile

```diff
@@ -27,15 +27,14 @@ SYSTEM_GROUP	 = http
 INSTALL_DEP_SERVICES = apache-web mysql
 START_DEP_SERVICES = apache-web mysql
 
-INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+INSTALL_PREFIX = /var/packages/$(SPK_NAME)/target
```

Comment:
you should remove the definition of `INSTALL_PREFIX`, it matches the default defined in `spksrc.directories.mk`